backend/side_stacker/game/models.py

```diff
@@ -188,12 +199,30 @@ def check_horizontal_connected(self, row, last_played_index, current_player):
     def place_movement(self, empty_index, row, row_num):
         row[empty_index] = self.current_player
         self.board[row_num] = row
-        if not self.check_winner(row_num, empty_index, self.current_player):
+        is_winner = self.check_winner(row_num, empty_index, self.current_player)
+        if not is_winner:
             self.toggle_current_player()
         self.save()
+        return is_winner
 
     def toggle_current_player(self):
         if self.current_player == 'X':
             self.current_player = 'O'
         else:
             self.current_player = 'X'
+
+    def find_random_empty_index(self):
```

Comment:
this function is the one responsible for playing the bot.
It is a simple function that uses choices from random package to guess empty places that the bot could run.